lib/cli/run-helpers.js

```diff
@@ -143,10 +143,8 @@ exports.handleFiles = ({
     try {
       newFiles = utils.lookupFiles(arg, extension, recursive);
     } catch (err) {
-      if (err.message.indexOf('cannot resolve path') === 0) {
-        console.error(
-          `Warning: Could not find any test files matching pattern: ${arg}`
-        );
+      if (err.code === 'ERR_MOCHA_NO_FILES_MATCH_PATTERN') {
+        console.warn(`Warning: ${err.message}: ${err.pattern}`);
```

Comment:
Something like `console.warn('Warning: %s: %O', err.message, err.pattern)` would also work, since Node.js' console methods run parameters thru `util.format`.  This would save the trouble of calling `JSON.stringify()` in a template string, anyhow.

Reply:
My concern was to ensure pattern would be displayed _exactly_ without evaluation. As long as that is addressed, don't care about method.